modular_skyrat/modules/altborgs/code/tallborg/robot.dm

```diff
@@ -18,6 +18,53 @@
 	else
 		regenerate_icons()
 
+// Brand stamps for cyborgs
+/mob/living/silicon/robot/examine(mob/user)
+	. = ..()
+
+	var/mob/living/silicon/robot/robot = src
+	if(R_TRAIT_TALL in robot.model.model_features)
```

Comment:
this is the snowflakey-ness that i'm talking about, it doesn't rely on a flag or anything, just a trait (exclusive to tallborgs)